Automattic/sensei
# Update naming from onboarding to setup wizard

Fixes #3215, Fixes #3179

### Changes proposed in this Pull Request

* Update naming from onboarding to setup wizard. 

### Testing instructions

* It changed a lot of Setup Wizard files. So it's good go through all the setup wizard to test. These changes should not affect any behavior. The new URL is `/wp-admin/admin.php?page=sensei_setup_wizard`
